cluster/addons/dns/kube2sky/kube2sky.go

```diff
@@ -58,6 +61,9 @@ var (
 	argDomain              = flag.String("domain", "cluster.local", "domain under which to create names")
 	argEtcdMutationTimeout = flag.Duration("etcd-mutation-timeout", 10*time.Second, "crash after retrying etcd mutation for a specified duration")
 	argEtcdServer          = flag.String("etcd-server", "http://127.0.0.1:4001", "URL to etcd server")
+	argEtcdCAFile          = flag.String("etcd-cafile", "", "Certificate Authority file to secure etcd communication")
+	argEtcdCertFile        = flag.String("etcd-certfile", "", "TLS certificate file to secure etcd communication")
+	argEtcdKeyFile         = flag.String("etcd-keyfile", "", "TLS key file to secure etcd communication")
```

Comment:
I'm already using/augmenting the existing `storage.etcd.EtcdConfig` - it's just that the config itself does not expose much. Functionality-wise, it would be great if it exposed `newClient` which I'm basically duplicating (together with `newTransport`)